src/javascripts/ng-admin/Main/component/service/config/Field.js

```diff
@@ -22,6 +22,7 @@ define(function (require) {
         format: 'yyyy-MM-dd',
         template: defaultValueTemplate,
         isDetailLink: false,
+        detailLinkRoute: '',
```

Comment:
just set the default value to 'edit' here and you won't need the addition line 50.